doc/readthedocs/elementsthermo.rst

```diff
@@ -3,6 +3,7 @@ Thermo Elements
 
 The only material, which is used for thermo elements in the input examples is
 
-- THERM_FourierIso
+- MAT_Fourier
+- MAT_Soret
 
-it seems.
\ No newline at end of file
+it seems.
```

Comment:
What is the point of "it seems"? Please remove it.

Reply:
Removed and description updated.